### PR TITLE
Update gitmojis.json and icons

### PR DESCRIPTION
**Pull request by meiMingle**
Update gitmojis.json and icons

### CODE REVIEW

**Comment by patou** [comment]
Hello,

Thanks for this PR.

File: src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt [hunk 1]
```diff
@@ -229,7 +229,7 @@ class GitCommitAction : AnAction() {
     private fun loadGitmojiFromHTTP() {
         val client = OkHttpClient().newBuilder().addInterceptor(SafeGuardInterceptor()).build()
         val request: Request = Builder()
-            .url("https://raw.githubusercontent.com/carloscuesta/gitmoji/master/src/data/gitmojis.json")
+            .url("https://raw.githubusercontent.com/carloscuesta/gitmoji/master/packages/gitmojis/src/gitmojis.json")
```
Comment (patou) [inline src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt:232]: There are now an api like describe in this readme :
https://github.com/carloscuesta/gitmoji/tree/master/packages/gitmojis

I think it's more good to use this url : https://gitmoji.dev/api/gitmojis


https://gitmoji.dev/api/gitmojis

**Review by patou — CHANGES_REQUESTED** [review]
Can you change the url to use the gitmoji api ?

Thanks for your contribution.

File: src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt (hunk 1 above)
Comment (meiMingle) [reply src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt:232]: Agree

**Review by patou — APPROVED** [review]
Thanks

**Pull request merged**